axelrod/strategies/prober.py

```diff
@@ -171,9 +173,61 @@ def strategy(self, opponent):
         # React to the opponent's last move
         if opponent.history[-1] == D:
             return D
-        # Otherwise cooperate, defect with a small probability
+        # Otherwise cooperate, defect with probability 1 - self.p
         choice = random_choice(1 - self.p)
         return choice
 
     def __repr__(self):
         return "%s: %s" % (self.name, round(self.p, 2))
+
+
+class RemorsefulProber(NaiveProber):
+    """
+    Like Naive Prober, but it remembers if the opponent responds to a random
+    defection with a defection by being remorseful and cooperating.
+
+    For reference see: "Engineering Design of Strategies for Winning
+    Iterated Prisoner's Dilemma Competitions" by Jiawei Li, Philip Hingston,
+    and Graham Kendall.  IEEE TRANSACTIONS ON COMPUTATIONAL INTELLIGENCE AND AI
+    IN GAMES, VOL. 3, NO. 4, DECEMBER 2011
+
+    A better description is given in the selfish gene:
+    https://books.google.co.uk/books?id=ekonDAAAQBAJ&pg=PA273&lpg=PA273&dq=remorseful+prober&source=bl&ots=kAeYRYg7GB&sig=RD5-XtDAxzTF9rxRZEWyFjwuKhc&hl=en&sa=X&ved=0ahUKEwiFg--H_qvNAhWXF8AKHQVTAzcQ6AEIKDAC#v=onepage&q=remorseful%20prober&f=false
```

Comment:
That url can be shortened to just https://books.google.co.uk/books?id=ekonDAAAQBAJ